### ISSUE TITLE
[Report] Games Freezing or Not Launching

### ISSUE THREAD

**Issue by ecameracci** [holoiso-eol/holoiso#373]
## Your Issue:
I have set up a fresh install of HoloISO and when launching some games, they either freeze or don't launch.

So far I've tested with Hitman 3 and Elden Ring. Hitman 3 freezes on the intro videos, and mangohud stops updating. Elden Ring displays a white screen (probably the Namco Bandai splash screen) and freezes, or is stuck at the spinning Steam logo.

With Hitman 3, launching in fullscreen mode from the desktop works, I was able to get in-game and run around for a bit. With Elden Ring, launching in desktop mode does not work either, even launching without the game overlay.

In both cases, I am still able to open the Steam menu and close the game from there, it is just the game itself that freezes.

I've attached logs from each game (launching in desktop mode with PROTON_LOG=1).

[logs.zip](https://github.com/theVakhovskeIsTaken/holoiso/files/9135253/logs.zip)

## Your device's full specs
```
OS: SteamOS Holo x86_64
Host: OptiPlex 3060
Kernel: 5.13.0-valve21-2-neptune-02209-g2a5bdc1102a0
Packages: 875 (pacman), 6 (flatpak)
Shell: bash 5.1.16
Resolution: 2560x1440
DE: Plasma 5.23.5
CPU: Intel i5-8400 (6) @ 4.000GHz
GPU: AMD ATI 03:00.0 Device 743f
GPU: Intel CoffeeLake-S GT2 [UHD Graphics 630]
Memory: 1816MiB / 7774MiB
```
The AMD GPU is a Sapphire RX 6400

## Your OS version (`cat /etc/os-release | grep VARIANT_ID | cut -d '"' -f 2`):
```
snapshot20220716.2021
```

## (Only for SteamOS Session loading issues) Have you tried launching `gamescope-session` from terminal? If yes, provide a log.
```
If useful, uploaded the output.
```
[gamescope-session.txt](https://github.com/theVakhovskeIsTaken/holoiso/files/9135242/gamescope-session.txt)

## Your installation type:
```
decksperience
```

## Have you updated your installation recently?
```
`steamos-update check && steamos-update now` reports "System up to date." as of submitting this ticket.
```

**Comment by ecameracci** [comment]
Elden Ring works fine if I launch before Hitman 3. It seems like Hitman 3 is the only one with issues or is maybe causing a process to hang. I launched Elden Ring after launching Hitman 3 and it getting stuck, and it did the "spinning steam logo" and restarted the gamescope session.

Playing around with the Hitman 3 settings, I was able to get in-game once by disabling vsync and using fullscreen instead of exclusive fullscreen, but the same freezing/crashing still happens after that.

Attaching a log of launching Hitman 3 from gamescope session and causing the gamescope session to restart:
[steam-1659040.zip](https://github.com/theVakhovskeIsTaken/holoiso/files/9143005/steam-1659040.zip)
